OpenScienceJournal/whistlepunk_library/src/main/java/com/google/android/apps/forscience/whistlepunk/NoteViewHolder.java

```diff
@@ -234,6 +234,9 @@ private static SensorAppearance getSensorAppearance(GoosciIcon.IconPath iconPath
             case GoosciIcon.IconPath.LEGACY_ANDROID_BLE:
                 return SensorTypeProvider.getSensorAppearance(Integer.valueOf(iconPath.pathString),
                         "");
+            case GoosciIcon.IconPath.MKRSCI_ANDROID_BLE:
+                return MkrSciBleSensorAppearance.get(iconPath.pathString);
+            // TODO
```

Comment:
Can you remove the TODO or add an explanation about what needs to be done?

Reply:
Removed.